docs/styles/custom.css

```diff
@@ -1897,8 +1897,8 @@ samp {
 code {
   padding: 2px 4px;
   font-size: 90%;
-  color: #c7254e;
```

Comment:
Oh! Bummer!
This is the wrong file Sun! You need to do this on /src/doc/template/static/styles, as this file is being generated!